p2p/node/pubsubManager/gossipsub.go

```diff
@@ -362,11 +362,25 @@ func (g *PubsubManager) ValidatorFunc() func(ctx context.Context, id p2p.PeerID,
 				backend.Logger().WithField("err", err).Warn("Sanity check of work object share view failed")
 				return pubsub.ValidationReject
 			}
-			if valid := backend.CheckIfValidWorkShare(block.WorkObjectHeader()); valid != types.Valid {
-				backend.Logger().Error("work share received from peer is not valid")
+
+			threshold := params.NewWorkSharesThresholdDiff
+			if block.NumberU64(common.ZONE_CTX) >= params.GoldenAgeForkNumberV3 {
+				threshold = backend.GetWorkShareP2PThreshold()
```

Comment:
Why only broadcast subshares after the height? They don't affect consensus, is this just to reduce noise?

Reply:
just to reduce the noise and so that people dont get deranked in gossipsub